cmd/machine-config-daemon/start.go

```diff
@@ -32,6 +33,7 @@ func init() {
 	startCmd.PersistentFlags().StringVar(&startOpts.kubeconfig, "kubeconfig", "", "Kubeconfig file to access a remote cluster (testing only)")
 	startCmd.PersistentFlags().StringVar(&startOpts.nodeName, "node-name", "", "kubernetes node name daemon is managing.")
 	startCmd.PersistentFlags().StringVar(&startOpts.rootMount, "root-mount", "/rootfs", "where the nodes root filesystem is mounted for chroot and file manipulation.")
+	startCmd.PersistentFlags().StringVar(&startOpts.onceFrom, "once-from", "", "Runs the daemon once using the provided URI as it's configuration source")
```

Comment:
Good catch. I'll clarify that this can be a path to a file or a URL. URI does make it sound like `http://`, `ftp://`, `file://`, etc.. would be supported when really it's `http[s]://` and `/...`.